partner_compassion/views/partner_compassion_view.xml

```diff
@@ -243,35 +234,24 @@
         <field name="inherit_id" ref="base.view_res_partner_filter"/>
         <field name="arch" type="xml">
             <field name="category_id" position="after">
-                <field name="lang"/>
-                <field name="city" filter_domain="['|',('city','ilike', self),('zip','ilike',self)]"/>
+                <field name="lang" />
+                <field name="city" filter_domain="['|',('city','ilike', self),('zip','ilike',self)]" />
```

Comment:
This change is producing your bug in Travis. It is probably due to a wrong merge during rebase.